src/services/messenger.py

```diff
@@ -403,7 +403,7 @@ def process_postback(messenger, payload):
             payload='INSTALL_GIT'
         )
         qr2 = quick_replies.QuickReply(
-            title=_('🤓 I\'ve Git Installed'),
+            title=_('🤓 I\'ve got Git Installed'),
```

Comment:
The title should not have more that [20 characters](https://developers.facebook.com/docs/messenger-platform/reference/buttons/quick-replies). Please restore the previous formulation!